discovery_engine_core/bindings/src/types/document/time_spent.rs

```diff
@@ -51,8 +51,19 @@ pub unsafe extern "C" fn time_spent_place_of_smbert_embedding(
 /// The pointer must point to a valid [`TimeSpent`] memory object, it
 /// might be uninitialized.
 #[no_mangle]
-pub unsafe extern "C" fn time_spent_place_of_time(place: *mut TimeSpent) -> *mut Duration {
-    unsafe { addr_of_mut!((*place).time) }
+pub unsafe extern "C" fn time_spent_place_of_view_time(place: *mut TimeSpent) -> *mut Duration {
```

Comment:
🔧 in l.47 (can't mark it here) the docs need to be changed as well to the new field name.